config_template.cfg

```diff
@@ -1473,7 +1855,73 @@ DV_SENS_FILENAME= surface_sensitivity.dat
 % as an ASCII file with name given by DV_SENS_FILENAMEand with format as
 % rows of x, y, z, dJ/dx, dJ/dy, dJ/dz for each grid point.
 DV_SENSITIVITY_FORMAT= SU2_NATIVE
+%
 DV_UNORDERED_SENS_FILENAME= unordered_sensitivity.dat
+%
+% Hold the grid fixed in a region
+HOLD_GRID_FIXED= NO
+%
+% Coordinates of the box where the grid will be deformed (Xmin, Ymin, Zmin, Xmax, Ymax, Zmax)
+HOLD_GRID_FIXED_COORDS= (-1e15, -1e15, -1e15, 1e15, 1e15, 1e15)
+%
+% Design variable for FEA problems
+% options: NONE, YOUNG_MODULUS, POISSON_RATIO, DENSITY, DEAD_WEIGHT, ELECTRIC_FIELD
+DESIGN_VARIABLE_FEA= NONE
+%
+% Penalty weight value to maintain the total sum of DV constant
+TOTAL_DV_PENALTY= 0.0
+%
+% Parameters for the corresponding OF (allowed stress and KS multiplier).
+STRESS_PENALTY_PARAM= (1.0, 10.0)
+%
+% Geometric conditions
+GEOMETRIC_CONDITIONS= SMALL_DEFORMATIONS
+%
+% Material model
+MATERIAL_MODEL= LINEAR_ELASTIC
+%
+% Compressibility of the material
+MATERIAL_COMPRESSIBILITY= COMPRESSIBLE
+%
+
+% ---------------- AUTOMATIC DIFFERENTIATION -------------------%
+%
+% Direct differentiation mode (forward)
+DIRECT_DIFF= NONE
+%
+% Automatic differentiation mode (reverse)
+AUTO_DIFF= NO
```

Comment:
We can remove this option (but keeping the bool in CConfig)

Reply:
do you want to phase out this keyword?

Reply:
Yes, setting this manually doesn't do anything

Reply:
I removed these in the config_template and in CConfig, but not anywhere else.